modules/dr-restoring-cluster-state.adoc

```diff
@@ -230,7 +410,7 @@ If the output includes multiple revision numbers, such as `2 nodes are at revisi
 +
 In a terminal that has access to the cluster as a `cluster-admin` user, run the following commands.
 
-.. Update the `kubeapiserver`:
+.. Force a new rollout for the Kubernetes API server:
```

Comment:
I would just double check that these steps to force these rollouts are still necessary, with the new step on creating new machines.

Reply:
@marun Should step 14 (and all the substeps) in this procedure be removed because these rollouts aren't necessary if the lost control plane hosts are being deleted and recreated?

Reply:
The forced rollouts are only necessary when one or more of the non-recovery control plane nodes could potentially have stale static pod content (i.e. the nodes were not created as part of the recovery procedure).